lib/services/models/commission.dart

```diff
@@ -63,8 +64,8 @@ class Commission {
   }
 
   String get formattedBondedTokens {
-    var tokens = bondedTokensCount.nhashToHash(fractionDigits: 7).split('.');
-    return '${tokens[0].formatNumber()}.${tokens[1]} HASH';
+    var tokens = bondedTokensCount.nhashToHash(fractionDigits: 7);
```

Comment:
I think `denom_util` should be used and `int_extension` removed. Seems like the `fractionalDigits` should be unified everywhere.